client/jetpack-cloud/sections/partner-portal/company-details-form/index.tsx

```diff
@@ -153,7 +156,39 @@ export default function CompanyDetailsForm( {
 						disabled={ isLoading }
 					/>
 				</FormFieldset>
-
+				<FormFieldset>
+					<FormLabel>{ translate( 'Which answer below best describes your company:' ) }</FormLabel>
+					<FormRadio
+						label={ translate( 'Agency' ) }
+						value="Agency"
+						checked={ companyType === 'Agency' }
+						onChange={ ( event: ChangeEvent< HTMLInputElement > ) =>
+							setCompanyType( event.target.value )
+						}
+						disabled={ isLoading }
+						className={ undefined }
+					/>
+					<FormRadio
+						label={ translate( 'Freelancer/Pro' ) }
+						value="Freelancer/Pro"
+						checked={ companyType === 'Freelancer/Pro' }
+						onChange={ ( event: ChangeEvent< HTMLInputElement > ) =>
+							setCompanyType( event.target.value )
+						}
+						disabled={ isLoading }
+						className={ undefined }
+					/>
+					<FormRadio
+						label={ translate( 'A business with multiple sites' ) }
+						value="A business with multiple sites"
+						checked={ companyType === 'A business with multiple sites' }
```

Comment:
I'm a bit skeptical about the value being a string here.

Reply:
I'm not doubting you of course, but could you elaborate on why a string with spaces might not be a string value?

Reply:
Well, there is not really an issue with this. It is obviously a personal choice, but I would prefer a value without spaces(more like an ID) rather than a string like this.

Reply:
I can understand that.  And it makes sense to me too.  I'll move the question back to the card to see if Simon is ok with that.